packages/ui/src/components/MenuV2/index.tsx

```diff
@@ -114,12 +114,12 @@ const FwdMenu = forwardRef(
 
       // Focus the first item when the menu is opened
       if (!isVisible) {
-        requestIdleCallback(() => {
+        setTimeout(() => {
           // We have to wait for the popup to be inserted in the DOM
           if (popupRef.current?.firstChild?.firstChild instanceof HTMLElement) {
             popupRef.current.firstChild.firstChild.focus()
           }
-        })
+        }, 1)
```

Comment:
Why providing 1? Can't we leave it empty (so it will be 0 by default) and so it will be executed asap

Reply:
It doesn't work with a timeout of `0`, not exactly sure why... Maybe due to how React schedules the update